starter/script.js

```diff
@@ -1 +1,33 @@
 // put code here!
+var tantrumFree = function() {
+  console.log("**************************")
+}
+console.log ("You've overslept one morning and realized you have 25 minutes to get yourself and your kindergartener out the door")
+var wakeUpInput = prompt("You can either jolt out of bed or push the snooze button") (Please enter 'j' or 's')
```

Comment:
Note to that if we want to use quotes *inside our string*, we have to use single quotes, because double quotes would *end* our string rather than be part of it.

(You got this correct, but I wanted to call it to your attention).